.claude/skills/rebase-conflicting-prs/SKILL.md

```
@@ -0,0 +1,47 @@
+---
+name: rebase-conflicting-prs
+description: Find every open PR assigned to or review-requested from the current user that has merge conflicts with master, rebase each one, force-push the update, and comment to request a fresh review from @claude and from the author. Use when the user wants to clear out PRs stuck on conflicts, review PRs that need rebase, or bulk-rebase their conflicting PR queue.
+---
+
+## Process
+
+### 1. Find candidate PRs
+
+Same two queries as the `review-queue` skill (the repo is resolved from `git remote -v`):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)